src/components/search-box/searchbox.module.scss

```diff
@@ -0,0 +1,29 @@
+.searchbox_container {
+  background-color: #fefeff;
+  opacity: 1;
+  width: 60%;
+  height: 100px;
+  border-radius: 20px;
+  box-shadow: 5px 5px 10px grey;
+  margin: 5% auto;
+}
+.button {
+  background-color: #382bd0;
+  color: white;
+  font-weight: bold;
+  width: 17%;
+  height: 60px;
+  margin: 20px 20px 2px 10px;
+  cursor: pointer;
+  border-radius: 5px;
+  border: none;
+}
+.searchbox {
+  background: #e4e5e4;
+  width: 75%;
+  height: 60px;
+  border-radius: 5px;
+  padding-left: 70px;
+  margin: 20px 20px 2px 10px;
+  border: none;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/63191499/165042540-c539c7c9-8563-45ca-a343-56f59647cf9a.png)

Can we make it responsive as well? Also, decreasing the size would make it blend in better, what do you think?

Reply:
ok I will work on it